uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt311/client/paho/Mqtt311ConnectionImpl.java

```diff
@@ -66,8 +67,8 @@ public ConnectResult start(MqttLib.ConnectionParams connectionParams, int connec
         this.connectionId = connectionId;
         try {
             MqttConnectOptions connectOptions = convertParams(connectionParams);
-            IMqttToken token = mqttClient.connectWithResult(connectOptions);
-            token.waitForCompletion(connectionParams.getConnectionTimeout());
+            IMqttToken token = mqttClient.connect(connectOptions);
```

Comment:
Looks like another paho API is used.
It is related to unsubscribe or fix existing bug?

Reply:
I changed sync client to async, because sync paho-client  hasn't response for unsubscribe method.